src/components/map-line/readme.md

```diff
@@ -0,0 +1,20 @@
+# gx-map-line
+
```

Comment:
A couple of lines explaining what the component does would be great!
Bonus points if you paste a usage example!

Reply:
Done. Addresed in 6aa1ac5